src/gs-page.c

```diff
@@ -481,6 +515,45 @@ gs_page_install_app (GsPage *page,
 	}
 }
 
+void
+gs_page_copy_app (GsPage *page,
+		  GsApp *app,
+		  const gchar *dest,
+		  GsShellInteraction interaction,
+		  GCancellable *cancellable)
+{
+	GsPagePrivate *priv = gs_page_get_instance_private (page);
+	GsPageHelper *helper;
+
+	helper = g_slice_new0 (GsPageHelper);
+	helper->action = GS_PLUGIN_ACTION_COPY;
+	helper->app = g_object_ref (app);
+	helper->page = g_object_ref (page);
+	helper->cancellable = g_object_ref (cancellable);
+	helper->interaction = interaction;
+
+	if (gs_app_get_state (app) == AS_APP_STATE_INSTALLED) {
```

Comment:
Use `gs_app_is_installed`. Otherwise, if the app is e.g. updatable, then it won't be able to copy it to the USB.

Reply:
Good catch. Fixed locally